daal4py/sklearn/ensemble/_forest.py

```diff
@@ -447,13 +452,21 @@ def _fit_regressor(self, X, y, sample_weight=None):
             FutureWarning
         )
 
-    daal_oob_score = \
-        self.oob_score and daal_check_version((2021, 'P', 500)) or self.oob_score is False
-    daal_ready = self.warm_start is False and \
-        self.criterion in ["mse", "squared_error"] and self.ccp_alpha == 0.0 and \
-        not sp.issparse(X) and daal_oob_score
-
-    if daal_ready:
+    _patching_status = PatchingConditionsChain(
+        "sklearn.ensemble.RandomForestRegressor.fit")
+    _dal_ready = _patching_status.and_conditions([
+        (self.oob_score and daal_check_version((2021, 'P', 500)) or not self.oob_score,

```

Comment:
As I know, oob_score is available since 2021.4 release

Reply:
This check was firstly introduced at https://github.com/intel/scikit-learn-intelex/commit/5875ea8f123d0f5f0d0d179b16dd161d670e9628#diff-d1633e1398300003a1cb5898a97cca1af6ef162666228ec14a94ae917003123eR302. So, it is available since 2021.5